ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
+    background-color: rgb(255, 255, 255);
+    height: 330px;
+    width: 99%;
+    position: relative;
+    display: flex;
+}
+
+.carousel p {
+    margin: 4px 0px;
+}
+
+.carousel img {
+    background-color: rgb(236, 236, 236);
+    height: 70%;
+    width: 98%;
```

Comment:
98? Why not 100%?

Reply:
Couldn't find another solution to separate the images (to make that grey line visible between them)

Reply:
reduce the img size or add padding